metered/src/atomic.rs

```diff
@@ -67,3 +77,15 @@ impl_blocks_for!(u16: serialize_u16);
 impl_blocks_for!(u32: serialize_u32);
 impl_blocks_for!(u64: serialize_u64);
 impl_blocks_for!(u128: serialize_u128);
+
+#[cfg(test)]
+mod tests {
+    #[test]
+    fn test_atomic_wraps() {
+        use std::sync::atomic::Ordering;
+        let a = atomic::Atomic::<u8>::new(255u8);
+
+        a.fetch_add(1, Ordering::Relaxed);
+        assert_eq!(a.load(Ordering::Relaxed), 0u8);
+    }
```

Comment:
Fair, this is using the `atomic` crate as a wrapper which does not make this claim, and I aimed to ensure a build breakage if that crate ever made a different choice (unlikely since its goal is just to provide a num type-generic wrapper over std)

Reply:
Oh, true. I guess that testing the `AtomicInt` would still be nice, but I can totally understand the point.

Reply:
Agreed, I'm now testing the behavior through `AtomicInt`